### PR TITLE
Bug 1601820 - Add pre-commit hooks for black

### PR DESCRIPTION
**Pull request by SuyashSalampuria**
Done in #6277

### CODE REVIEW

**Comment by SuyashSalampuria** [comment]
@klahnakoski @armenzg 
I have configured black for being used in treeherder
Currentlyy I have ingnored only the migration files.

**Comment by klahnakoski** [comment]
@SuyashSalampuria 

This is good except that `isort` and `black` are fighting for dominance!

```
C:\Users\kyle\code\treeherder>git commit -m "black formatting
flake8...................................................................Passed
isort....................................................................Failed
- hook id: isort
- files were modified by this hook

Fixing treeherder\extract\extract_alerts.py

shellcheck...........................................(no files to check)Skipped
prettier.............................................(no files to check)Skipped
black....................................................................Failed
- hook id: black
- files were modified by this hook

reformatted treeherder\extract\extract_alerts.py
All done! \u2728 \U0001f370 \u2728
1 file reformatted.
```

You must remove `isort`

**Comment by SuyashSalampuria** [comment]
@klahnakoski Done the changes

**Comment by armenzg** [comment]
Hi @SuyashSalampuria there's some conflicts that need to be solved.
I will let Kyle handle the review.
I will merge it once you approve it.

File: requirements/dev.in [hunk 1]
```diff
@@ -15,6 +15,7 @@ seed-isort-config
 flake8
 isort
```
Comment (klahnakoski) [inline requirements/dev.in:16]: Remove everything related to `isort`, especially if it is running in Travis

**Pull request closed without merging**